tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -116,10 +116,6 @@ describe('DataAdapterMixin', () => {
 
           expect(sessionService.invalidate).to.have.been.calledOnce;
         });
-
-        it('returns true', () => {
-          expect(adapter.handleResponse(401)).to.be.true;
-        });
```

Comment:
Shouldn't this be replaced with below's `it("returns _super's return value", () => {` as well?